web/source/osk/embedded/subkeyDelegator.ts

```diff
@@ -0,0 +1,19 @@
+namespace com.keyman.osk.embedded {
+  export class SubkeyDelegator {
```

Comment:
What's the difference between the `SubkeyDelegator` and the `SubkeyPopup`? Why are both needed? `SubkeyDelegator` seems to be a subset of `SubkeyPopup`?

Reply:
With regard to the description, `SubkeyDelegator` becomes the embedded implementation of `RealizedGesture` for subkeys, while `SubkeyPopup` becomes the in-browser (site-integrated) implementation of `RealizedGesture`.

Reply:
Can you add comments to the code accordingly? I understand the names now but I had no idea what `SubkeyDelegator` was delegating to at first 😆